test/e2e/config/ibmcloud-e2e-powervs.yaml

```diff
@@ -10,23 +10,23 @@ providers:
     type: CoreProvider
     versions:
     - name: v1.1.4
```

Comment:
you may need to change version here as well and all the remaining places in the version section?

Reply:
sorry, missed it. Updated.